calico/networking/configuring/qos-controls.mdx

```diff
@@ -0,0 +1,91 @@
+---
+description: Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections of Calico workloads.
+---
+
+# Configure QoS Controls
+
+## Big picture
+
+Configure QoS (Quality of Service) Controls to limit ingress and/or egress bandwidth, packet rate and number of connections used by Calico workloads in order to prevent them from overusing or overloading network resources.
+
+## Value
+
+With QoS controls, $[prodname] enforces network resource limits (bandwidth, packet rate, connections) for Kubernetes pods and OpenStack VMs, ensuring fair resource allocation and preventing performance degradation for other workloads.
+
+## Concepts
+
+### Quality of Service Controls
+
+**Quality of Service**, or **QoS** for short, in the field of computer networking, refers to traffic prioritization and resource reservation control mechanisms. It may either mean guaranteeing minimum levels of networking resources, or limiting the maximum levels of usage of networking resources in order to prevent it from being overloaded by a small number of actors. For practical purposes, the latter will be considered as $[prodname] implements QoS Controls in this manner.
+
+### Annotations
+
+On Kubernetes, $[prodname] uses annotations on pods for the configuration of QoS controls. You can annotate pods in any of these ways:
+- pods can be directly annotated with `kubectl annotate`;
+- they can be created with the annotations in their metadata;
+- if they're part of a deployment or daemonset, they can have the annotations in the metadata of their template spec.
+
+The values of the QoS controls annotations are parsed using the [Quantity type of the resource package](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity),
+so they may contain suffixes, such as `k`, `M` or `G` (`1k` represents "1000", `1Ki` represents "1024", and so on).
```

Comment:
Again, too early. Does not seem like a user need to know what package parses the annotations, what is important is to say (at the right place) what is the syntax. What you so not say is whether it is in bytes or bits. My assumption would be bits per second since it is bandwidth.

Reply:
This is before the specific types of controls, so I wanted to only mention the quantity suffixes (but yes, for bandwidth it will be bits per second, for packet rate, packets per second and for connections an absolute number). I'll remove the mention to the package the parses and just explain the suffixes/syntax...